include/LightGBM/config.h

```diff
@@ -806,6 +807,14 @@ struct Config {
   // desc = when ``multi_error_top_k=1`` this is equivalent to the usual multi-error metric
   int multi_error_top_k = 1;
 
+  // type = multi-double
+  // default = None
+  // desc = list representing flattened matrix (in row-major order) giving loss weights for classification errors
+  // desc = list should have ``n * n`` elements, where ``n`` is the number of classes
+  // desc = the matrix co-ordinate ``[i, j]`` should correspond to the ``i * n + j``-th element of the list
+  // desc = if not specified, will use equal weights for all classes
+  std::vector<double> auc_mu_weights;
```

Comment:
After removing example file with weights, I expect some issues in the future with questions about how to use `auc_mu_weights`. I'm not saying that the description is not clear, it's perfect! But still, it'll be nice to have an example we can refer as an answer to those questions.

Can we append `auc_mu` here 
https://github.com/microsoft/LightGBM/blob/6129208006c91f9ac94891690aca6f0c49d9b08e/examples/multiclass_classification/train.conf#L24
and add `auc_mu_weights` example in that config with original matrix view in comments like
```
#  0  1  2  3  4
#  5  0  6  7  8
#  9 10  0 11 12
# 13 14 15  0 16
# 17 18 19 20  0
auc_mu_weights = 0,1,2,3,4,5,0,6,7,8,9,10,0,11,12,13,14,15,0,16,17,18,19,20,0
```
?